### PR TITLE
Fixed bar chart, None is not shown anymore #86

### PR DESCRIPTION
**Pull request by Sebastiaanvm**
merge into master

### CODE REVIEW

File: src/components/Graphs/BarChart.js [hunk 1]
```diff
@@ -7,20 +7,20 @@ import React from 'react';
...
-            backgroundColor: 'rgba(255,99,132,0.2)',
-            borderColor: 'rgba(255,99,132,1)',
-            borderWidth: 1,
-            hoverBackgroundColor: 'rgba(255,99,132,0.4)',
-            hoverBorderColor: 'rgba(255,99,132,1)',
-            data: [65, 59, 80, 81, 56, 55, 40]
-        }
-    ]
-};
+// const data = {
```
Comment (emilhaugberg) [inline src/components/Graphs/BarChart.js]: delete this

Comment (Sebastiaanvm) [reply src/components/Graphs/BarChart.js]: done

File: src/components/Graphs/BarChart.js [hunk 2]
```diff
@@ -46,14 +31,16 @@ class BarChart extends React.Component {
...
     }
 
     makeData() {
         var labelArray = []
         var dataArray = []
-        this.state.delays.forEach(item => labelArray.push(Object.keys(item)[0]));
-        this.state.delays.forEach(item => dataArray.push(Object.values(item)[0]));
+        /* Put the label of the fetch data in the label array of the graph.
+         * Put the data of the fetch data in the data array of the graph. */
+        this.state.delays.forEach(item => labelArray.push(item['metric']['district']));
```
Comment (man-netcat) [inline src/components/Graphs/BarChart.js:45]: use map here

**Review by JoeyHen — APPROVED** [review]
idk man

**Pull request merged**